src/module-elasticsuite-core/Setup/ConfigOptionsList.php

```diff
@@ -38,17 +38,17 @@ class ConfigOptionsList implements ConfigOptionsListInterface
      * Input key for the options
      */
     const INPUT_KEY_ES_HOSTS = 'es-hosts';
-    const INPUT_KEY_ES_SSL   = 'es-enable-ssl';
-    const INPUT_KEY_ES_USER  = 'es-user';
-    const INPUT_KEY_ES_PASS  = 'es-pass';
+    const INPUT_KEY_ES_SSL = 'es-enable-ssl';
```

Comment:
Don't change this cosmetic, we like things aligned, and changing these lines is not related to your PR.